.github/release.yml

```diff
@@ -0,0 +1,27 @@
+# GitHub Auto-Generated Release Notes Configuration for RAPIDS
+# This file configures how GitHub automatically generates release notes
+
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+      - dependencies
+    authors:
+      - rapids-bot[bot]
+      - dependabot[bot]
+  categories:
+    - title: 🚨 Breaking Changes
+      labels:
+        - breaking
```

Comment:
For any repos using the Label Checker (https://docs.rapids.ai/resources/label-checker/), `breaking` is not mutually exclusive with these other options.

If I label a PR `breaking` AND `improvement`, what section(s) will it appear in? Both?

Reply:
Answered James in slack: https://nvidia.slack.com/archives/C01PF84NXCL/p1758115849630449?thread_ts=1758053969.214199&cid=C01PF84NXCL